kubernetes-client/pom.xml

```diff
@@ -170,7 +170,7 @@
     <dependency>
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>okhttp-ws</artifactId>
-      <version>${okhttp.version}</version>
+      <version>3.4.1</version>
```

Comment:
I also think that okhttp-ws might not be needed anymore as it has been added to okhttp directly.